controller/controller_server/csi_controller_server.py

```diff
@@ -106,8 +103,8 @@ def CreateVolume(self, request, context):  # pylint: disable=too-many-branches
                     logger.debug(
                         "volume was not found. creating a new volume with parameters: {0}".format(request.parameters))
 
-                    array_mediator.validate_supported_capabilities(capabilities)
-                    volume = array_mediator.create_volume(volume_final_name, size, capabilities, pool)
+                    array_mediator.validate_supported_capabilities(space_efficiency_type)
+                    volume = array_mediator.create_volume(volume_final_name, size, space_efficiency_type, pool)
```

Comment:
I think the 3rd argument here is currently defined to be a dict, not a string

Reply:
actually non-None wont be good cause we testing on xiv. fixed it on al mediators

Reply:
xiv or not, it doesn't matter - a space efficiency value should be passed from the request to the mediator, and this should be asserted in a controller server unit test

Reply:
done